docs/examples/frameworks/jax/flax-basic_example.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>Suggestion:</p><p>"Next step is to instantiate pipelines and build them." -&gt; "Next step is to instantiate DALI pipelines and build them." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4978/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done